zrml/market-commons/src/tests.rs

```diff
@@ -43,6 +43,7 @@ const MARKET_DUMMY: Market<AccountIdTest, BlockNumber, Moment> = Market {
     resolved_outcome: None,
     scoring_rule: ScoringRule::CPMM,
     status: MarketStatus::Disputed,
+    bonds: MarketBonds { advisory: None, oracle: None, validity: None },
```

Comment:
Can't call this here because the constant is evaluated at compile time, but `default()` is evaluated at runtime, similar to `static` in C++. If you're asking yourself why `default()` isn't declared `const`... well, you're not alone: https://stackoverflow.com/questions/72467891/why-isnt-defaultdefault-a-constant-function